src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/JdbcUtils.scala

```diff
@@ -0,0 +1,19 @@
+package com.microsoft.sqlserver.jdbc.spark.utils
+
+import org.apache.spark.sql.execution.datasources.jdbc.JDBCOptions
+import org.apache.spark.sql.jdbc.JdbcDialects
+
+import java.sql.Connection
+
+object JdbcUtils {
+  /**
+   * Creates a JDBC connection using the input JDBC options.
+   * @param options The options which are used to create the connection.
+   * @return A JDBC connection.
+   */
+  def createConnection(options: JDBCOptions): Connection = {
+    val dialect = JdbcDialects.get(options.url)
+    val conn = dialect.createConnectionFactory(options)(-1)
+    conn
+  }
+}
```

Comment:
I would recommend only modify the createConnectionFactory instead of adding a new JdbcUtils.scala, as there is only one method, and we could use two lines or even one line. Something like below,
val dialect = JdbcDialects.get(options.url)
val conn = dialect.createConnectionFactory(options)(-1)

Reply:
I was trying to be DRY, but if you prefer it I can change it. My thought was that a central place to create the connection would also help us in the future to apply changes. And I think the `JbdbUtils` name makes sense because in Spark they use the same: https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JdbcUtils.scala

Reply:
Any thoughts on this @shivsood ?

Reply:
Spark JdbcUtils.scala defines methods such as truncateTable, getSchema,  savePartition, similiar to [BulkCopyUtils.scala](https://github.com/microsoft/sql-spark-connector/blob/master/src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala) for Spark MsSQL Connector.
We could have a method for creating connection, but have a new file only for this connection is way too much. I could not think of any other methods in connector jdbcUtils.scala except we change the entire structure.